fuzz/Cargo.toml

```diff
@@ -20,6 +20,9 @@ features = ["fuzz"]
 path = "../ntp-daemon"
 features = ["fuzz"]
 
+[dependencies.pretty_assertions]
+version = "1.3.0"
+
```

Comment:
Do we really need this additional crate? I know we have strayed quite a bit from the original target of keeping the number of dependencies down, but this feels somewhat superfluous to me.

Reply:
it's quite difficult when you see a list of 100+ numbers to figure out where the error is. Is it needed? maybe not, but this is in our fuzzing, where I don't think it does any harm and is occasionally convenient.